laravel/app/TestHelpers.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+//NOTE: helper functions should be in "app", but not it's namespace
```

Comment:
Is there another place this file can live in? Like the tests folder? I'm assuming you should be able to put this anywhere, since it's being manually defined in the `composer.json` file. Not sure why, it just seems out of place living in the `app` folder all by itself. :thinking: